packages/account-v2/src/modules/IDVForm/index.tsx

```diff
@@ -0,0 +1,159 @@
+import React, { Fragment, useCallback, useEffect, useMemo, useState } from 'react';
+import { Field, FieldProps, FormikProps, useFormikContext } from 'formik';
+import { ResidenceList } from '@deriv/api-types';
+import { WalletDropdown } from '../../components/base/WalletDropdown';
+import { WalletTextField } from '../../components/base/WalletTextField';
+import { ResponsiveWrapper } from '../../components/responsive-wrapper';
+import { DOCUMENT_LIST } from '../../mocks/idv-form.mock';
+import { getIDVNotApplicableOption } from '../../utils/default-options';
+
+type TIDVFormProps = {
+    allowIDVSkip?: boolean;
+    selectedCountry: ResidenceList[0];
+};
+
+type TDocument = {
+    additional?: {
+        display_name?: string;
+        example_format?: string;
+    };
+    example_format?: string;
+    id: string;
+    text: string;
+    value?: string;
+};
+
+type TIDVFormValues = {
+    document_additional?: string;
+    document_number: string;
+    document_type: TDocument;
+    error_message?: string;
+};
```

Comment:
I thought we were supposed to use camelCase and in the case of a request payload we can create a payload from the values and send the values to the API

Reply:
Please post the same in the channel. Lets take the opinion of the others